operations/service_web.go

```diff
@@ -134,12 +134,19 @@ func startWebService() cli.Command {
 	}
 }
 
-func gracefulShutdownForSIGTERM(ctx context.Context, servers []*http.Server, wait chan struct{}, catcher grip.Catcher) {
+func gracefulShutdownForSIGTERM(ctx context.Context, servers []*http.Server, wait chan struct{}, catcher grip.Catcher, env evergreen.Environment) {
 	defer recovery.LogStackTraceAndContinue("graceful shutdown")
 	sigChan := make(chan os.Signal, len(servers))
 	signal.Notify(sigChan, syscall.SIGTERM)
 
 	<-sigChan
+	// we got the signal, so modify the status endpoint and wait
+	err := env.SetShutdown()
+	if err != nil {
+		grip.Error(err)
+	}
+
+	time.Sleep(10 * time.Second)
```

Comment:
I'm a bit torn about whether this should be configurable. Certainly in the long-term it should be, since users might want to change this value. Could you either make this configurable or open a ticket for it?

Edit: On second thought, I can imagine us wanting to have this set to a shorter value in staging.

Reply:
OK will make this configurable in the admin settings dash